packages/core/src/expression-builders/identifier.ts

```diff
@@ -0,0 +1,33 @@
+import { BaseSqlExpression } from './base-sql-expression.js';
+
+/**
+ * Use {@link @sequelize/core.identifier} instead.
```

Comment:
Why specifically `@sequelize/core.identifier`?

Reply:
For some reason typedoc can't find `identifier()` if I don't specify it's package :(